WalletWasabi.Tests/UnitTests/TransactionFactoryTests.cs

```diff
@@ -0,0 +1,441 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using WalletWasabi.Exceptions;
+using WalletWasabi.KeyManagement;
+using WalletWasabi.Models;
+using WalletWasabi.Models.TransactionBuilding;
+using WalletWasabi.Services;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests
+{
+	public class TransactionFactoryTests
+	{
+		[Fact]
+		public void InsufficientBalance()
+		{
+			var transactionFactory = CreateTransactionFactory(new[]{
```

Comment:
```suggestion
			var transactionFactory = CreateTransactionFactory(new[]
			{
```
this should be done across the whole PR. I can point them out all if you want.

Reply:
You can make pull requests btw to this branch: https://github.com/lontivero/Wasabito/tree/Refactoring/Extract-Transaction-Building

Reply:
fixed